### PR TITLE
Organized the Activity string resources 

### PR DESCRIPTION
**Pull request by jaimehta19**
## Explanation

In this PR, I've arranged HelpAcctivity string resources in the strings.xml file. I've also arranged FAQ string resources in a subheading of HelpActivity.

## Checklist

- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by jaimehta19** [comment]
@rt @FareesHussain @BenHenning  Please review my Pull Request

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -446,6 +445,22 @@
...
+  <string name="developer_options_mark_chapters_completed">Mark Chapters Completed</string>
+  <string name="developer_options_mark_stories_completed">Mark Stories Completed</string>
+  <string name="developer_options_mark_topics_completed">Mark Topics Completed</string>
+  <string name="developer_options_view_logs_label">View Logs</string>
+  <string name="developer_options_event_logs">Event Logs</string>
+  <string name="developer_options_override_app_behaviors_label">Override App Behaviors</string>
+  <string name="developer_options_show_all_hints_solution">Show all hints/solution</string>
+  <string name="developer_options_force_network_type">Force network type</string>
+  <string name="developer_options_force_crash">Force crash</string>
+  <!--  Help Activity-->
```
Comment (rt4914) [inline app/src/main/res/values/strings.xml:460]: Nit: `<!-- Help Activity -->`

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -52,7 +52,6 @@
   <string name="state_continue_submit">Submit</string>
   <string name="previous_state_description">Navigate to the previous card</string>
   <string name="next_state_description">Navigate to the next card</string>
-  <string name="state_submit_button">Submit</string>
```
Comment (rt4914) [inline app/src/main/res/values/strings.xml:55]: Why is this removed?

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -446,6 +445,22 @@
   <string name="ratio_default_hint_text" description="The default hint text for the ratio interaction if the placeholder text is not available.">
     Enter a ratio in the form x:y.
   </string>
+  <!-- DeveloperOptionsFragment -->
+  <string name="developer_options_activity_title">Developer Options</string>
```
Comment (rt4914) [inline app/src/main/res/values/strings.xml:449]: How does this PR help the codebase? Asking this because I don't see any usage of these strings in project which means that they are unused and not required.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@jaimehta1912 PTAL
